DeprecationWarning: Version._version is private and will be removed soon

With `packaging==26.0` borg emits a warning:

```
 .venv/lib/python3.14/site-packages/borg/__init__.py:9: in <module>
    __version_tuple__ = _v._version.release
                        ^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = (<Version('1.4.3')>,), kwargs = {}

    @functools.wraps(arg)
    def wrapper(*args, **kwargs):
>       _wm.warn(msg, category=category, stacklevel=stacklevel + 1)
E       DeprecationWarning: Version._version is private and will be removed soon
```

--- a/src/borg/__init__.py
+++ b/src/borg/__init__.py
@@ -5,8 +5,7 @@ from packaging.version import parse as parse_version
 from ._version import version as __version__
 
 
-_v = parse_version(__version__)
-__version_tuple__ = _v._version.release
+__version_tuple__ = parse_version(__version__).release
 
 # assert that all semver components are integers
 # this is mainly to show errors when people repackage poorly

